factory/processing/blockProcessorCreator.go

```diff
@@ -441,6 +436,38 @@ func (pcf *processComponentsFactory) newShardBlockProcessor(
 	return blockProcessorComponents, nil
 }
 
+func (pcf *processComponentsFactory) createBlockProcessor(
+	argumentsBaseProcessor block.ArgBaseProcessor,
+	validatorStatisticsProcessor process.ValidatorStatisticsProcessor,
+) (process.BlockProcessor, error) {
+	argShardProcessor := block.ArgShardProcessor{
+		ArgBaseProcessor: argumentsBaseProcessor,
+	}
+
+	shardProcessor, err := block.NewShardProcessor(argShardProcessor)
+	if err != nil {
+		return nil, errors.New("could not create block statisticsProcessor: " + err.Error())
```

Comment:
there is an old part of the log string here: `statisticsProcessor` should be removed

Reply:
Fixed